web/src/components/buttons/DownloadCsv.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from 'components/Button';
+import { Download } from 'lucide-react';
+import { useParams } from 'react-router-dom';
+import { DEFAULT_ICON_SIZE } from 'utils/constants';
+
+import { getCountryName, getZoneName } from '../../translation/translation';
+
+function getCountryToDownload(zoneId?: string) {
+  if (!zoneId) {
+    return '';
+  }
+
+  const zoneName = getZoneName(zoneId);
+  const zoneCountryName =
+    zoneId && zoneId.includes('-') && getCountryName(zoneId)?.toLowerCase();
+  return zoneCountryName || zoneName;
+}
```

Comment:
This don't really work as it gets the translated name of the country. So it breaks for everything except English.

Reply:
Resolved by changing to zoneIds